prow/config.yaml

```diff
@@ -1447,6 +1447,49 @@ postsubmits:
           - name: cache-ssd
             hostPath:
               path: /mnt/disks/ssd0
+      - name: ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7-init
+        agent: kubernetes
+        spec:
+          containers:
+          - image: gcr.io/k8s-testimages/e2e-kubeadm:v20170908-a7a2772e
```

Comment:
@krzyzacy I guess you have to bump the image here and push it to pick up the new changes, right?

Reply:
yes - we can make a separate PR